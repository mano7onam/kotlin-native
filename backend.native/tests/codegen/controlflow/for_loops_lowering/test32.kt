package codegen.controlflow.for_loops_lowering.test32

import kotlin.test.*

@Test fun runTest() {
	val arr = arrayOf(7, 10, 2, 6, 7, 9, 0, 4)
	for (i in arr) {
    if (i in arr.indices) {
      println(i)
    }
  }
}
