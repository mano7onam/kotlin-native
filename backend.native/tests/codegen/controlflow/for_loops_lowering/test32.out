7
2
6
7
0
4
